ftl-build/x86_64-musl/Dockerfile

```diff
@@ -23,7 +23,8 @@ RUN apk add --no-cache \
         jq \
         m4 \
         python3 \
-        py3-yaml
+        py3-yaml \
+        python3-requests
```

Comment:
Looks like you want `py3-requests`

https://pkgs.org/search/?q=py3-requests

https://pkgs.org/search/?q=python3-requests